src/main/java/com/zakem/RecipeCatalog/object/Ingredient.java

```diff
@@ -0,0 +1,31 @@
+package com.zakem.RecipeCatalog.object;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name="INGREDIENT")
+public class Ingredient {
+
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	Integer id;
+	
+	@Column(name="NAME")
```

Comment:
By default the Column name is the name of the variable, so you don't need this annotation when the two are the same.